CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.0.0 - 2022-12-22
+
+* Bump jsonwebtokens from 8.5.1 to 9.0.0 to mitigate CVE-2022-23529.
```

Comment:
Optional: I wondered whether we should indicate that we are mitigating the CVE even though we are unaffected, just so we don't worry anyone if they see a changelog about mitigating a CVE... Equally though recognise that we like to keep these brief. I'm not sure what is the best way to go but I lean towards mentioning that we were not affected but choose to upgrade anyway

Reply:
Sure, I will add an explainer 👍  Kept it brief for the reason you mentioned but let's try to minimise any alarm caused, that's probably good.